### ISSUE TITLE
Guide: Using blurhash to improve image display

### ISSUE THREAD

**Issue by react-learner** [meilisearch/documentation#622]
Short guide showing how you can use https://blurha.sh hashes for a prettier way to display images before they load in.

**Comment by guimachiavelli** [comment]
Not sure I understand this. How is it related to Meilisearch?

**Comment by dichotommy** [comment]
> Not sure I understand this. How is it related to Meilisearch?

If you have a dataset that involves images (e.g. movie posters in movies.json) then loading and displaying those images can become a problem when returning search results (e.g. blank spaces where images should be; results moving around the page as images load in; etc.). To avoid users implementing sloppy-looking search experiences w/ Meilisearch, we thought about providing a guide to improve image display and load-in using a tool like blurhash.

**Comment by guimachiavelli** [comment]
Hmmmm…yeah, I understand what you mean. 

I definitely think this is a better match for the resource center than the docs. Thinking as a dev, I really wouldn't go to a search engine's docs to find info on how to handle an issue that relates to building the front-end.

**Comment by ferdi05** [comment]
I agree with you @guimachiavelli . We'd be happy to take this over. I'll start working on the resource center in the coming days.
If this sounds good to you @dichotommy I think that you can transfer the issue to the DevRel repo.

**Comment by guimachiavelli** [comment]
@ferdi05, I don't think we have the correct permissions for transferring issues into meilisearch/devrel.

Can you or someone from the @meilisearch/devrel-team transfer it?
